### PR TITLE
Feature/additionalinputs

### PR DESCRIPTION
**Pull request by andy-schwedler**
Hi there,

In this PR, I was focussed on adding additional input fields for the beaver to input. 

Along the way I have realised that the data structure, as well as the UX could be improved. Although I had to touch so many files, you can only focus on the most relevant files for this [US](https://github.com/andy-schwedler/capstone-project/issues/10):

- [CREATE PAGE].js - Create a new Memory is no longer a Pop Up section, but a own page now
- [HEADER.test.js].js - adding a component test to the header
- [MEMORY CARD DETAIL].js - adding additional information from create-a-memory-form
- [_APP.js].js - changes on how a new MemoryObject is formed

Happy to hear from you, on how to improve this code.

Thanks a lot!

### CODE REVIEW

File: components/MemoryCard/MemoryCardDetail.js [hunk 1]
```diff
@@ -23,18 +23,8 @@ export default function MemoryCardDetail({ currentEvent, onToggleFavorite }) {
...
-            remaining essentially unchanged. It was popularised in the 1960s
-            with the release of Letraset sheets containing Lorem Ipsum passages,
-            and more recently with desktop publishing software like Aldus
-            PageMaker including versions of Lorem Ipsum.
-          </p>
+          <h3>{currentEvent.headline}</h3>
+          <p>{currentEvent.details}</p>
         </StyledTextFormatter>
       </StyledDetailFrame>
     </>
```
Comment (KirstenSchuhmann) [inline components/MemoryCard/MemoryCardDetail.js]: Den Wrapper <> kannst du in diesem Fall entfernen, da dein article bereits den Inhalt umfasst.

Comment (andy-schwedler) [reply components/MemoryCard/MemoryCardDetail.js]: good point 👍 thx

File: components/Forms/CreateMemoryForm.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  padding: 0.5em;
+`;
+const StyledFieldset = styled.fieldset`
+  display: flex;
+  flex-direction: column;
+  gap: 1.5em;
+  border: 1px solid var(--beaver3);
+  color: var(--beaver);
+
+  input {
```
Comment (julialoeschel) [inline components/Forms/CreateMemoryForm.js]: all the inputs are in MagicInput, I guess this and the following is unnecessary here now?

File: pages/bookmarks/index.js [hunk 3]
```diff
@@ -27,8 +20,8 @@ export default function Bookmarks({
     </Fragment>
   ));
 
-  const BookmarkListWithFallback =
-    favEvents?.length === 0 ? (
+  const favList =
+    favMemories?.length === 0 ? (
       <DisplayMessage message={"🥕 🦫 🥦"} />
```
Comment (julialoeschel) [inline pages/bookmarks/index.js:25]: haha :D

File: pages/createpage/index.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Footer from "../../components/Footer/Footer";
+import CreateMemoryForm from "../../components/Forms/CreateMemoryForm";
+import { StyledGridWrapper, StyledMain } from "../../components/GlobalStyles";
+import Header from "../../components/Header/Header";
+
+export default function CreatePage({ sampleEvents, onAddCreateCard }) {
+  return (
+    <>
```
Comment (julialoeschel) [inline pages/createpage/index.js]: your StyledGridWrapper is also a container - you don´t really need the Fragments here

**Review by julialoeschel — COMMENTED** [review]
Hey there. 
good work. 
I have almost nothing to add.

keep on going!!
:octocat:

File: pages/createpage/index.js (hunk 4 above)
Comment (andy-schwedler) [reply pages/createpage/index.js]: Thanks, good point

File: components/Forms/CreateMemoryForm.js (hunk 2 above)
Comment (andy-schwedler) [reply components/Forms/CreateMemoryForm.js]: Another good point. Question:

I wanted to keep the MagicInput as simple as possible with very little styling. 

In this Case MagicInput is called in the CreateForm. I have also called it in the Edit Form, but with a bit different styling.

Is that adequate or illegal? Happy to hear your thoughts on this topic

Comment (julialoeschel) [reply components/Forms/CreateMemoryForm.js]: Its late now, but yes, its perfectly fine like this .

**Pull request merged**